src/main.rs

```diff
@@ -133,22 +134,28 @@ fn main() -> Result<(), Box<dyn Error>> {
                 let rec = bam_paths.entry(c[0].to_owned()).or_insert_with(Vec::new);
                 rec.push((c[1].to_owned(), b[1].to_owned()))
             }
-
-            sample_calls.par_iter().for_each(|(sample, sample_call)| {
-                bcf::report::table_report::table_report(
-                    sample_call,
-                    fasta_path,
-                    bam_paths
-                        .get(sample)
-                        .unwrap_or_else(|| panic!("No bam provided for sample {}", sample)),
-                    output_path,
-                    sample,
-                    infos.clone(),
-                    formats.clone(),
-                    max_read_depth,
-                    js_file_names.clone(),
-                )
-                .unwrap_or_else(|_| panic!("Failed building table report for sample {}", sample));
+            let pool = rayon::ThreadPoolBuilder::new()
+                .num_threads(threads)
+                .build()?;
```

Comment:
You can also specify the number of threads for the global threadpool, which I think I'd prefer here since there's no point in managing multiple pools
```rust
rayon::ThreadPoolBuilder::new()
    .num_threads(threads)
    .build_global()?;
```

Reply:
(i.e. you can omit the whole `pool.install` part then)